data/data/powervs/cluster/power_network/pi_network.tf

```diff
@@ -12,15 +12,17 @@ resource "ibm_pi_dhcp" "new_dhcp_service" {
   count                  = var.pvs_network_name == "" ? 1 : 0
   pi_cloud_instance_id   = var.cloud_instance_id
   pi_cloud_connection_id = data.ibm_pi_cloud_connection.cloud_connection.id
+  pi_dns_server          = "1.1.1.1"
```

Comment:
I prefer avoiding this for long run, if its workaround at the moment - lets mention in the comment as TODO to remove once the issue gets fixed in the infrastructure.

Reply:
@mkumatag In the future, once Private Cluster Support lands, we will always be passing `pi_dns_server` and it will default to `1.1.1.1`.

Do you still think we need a `TODO`?

Reply:
what happens if we don't pass pi_dns_server? the api itself doesn't still default it to 1.1.1.1?

Reply:
> @mkumatag In the future, once Private Cluster Support lands, we will always be passing `pi_dns_server` and it will default to `1.1.1.1`.
> 
> Do you still think we need a `TODO`?

nope, we should have if/else sort of a block where if user supplies that thens set this param or else drop(because this is an optional param and powervs code should take care of setting right dns server) to rely on what powervs dhcp service gives.

Reply:
> what happens if we don't pass pi_dns_server? the api itself doesn't still default it to 1.1.1.1?

This is not happening with the latest code change but eventually that's the expectation